security/gnupg-pkcs11-scd/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+
+name                    gnupg-pkcs11-scd
+version                 0.10.0
+
+categories              security
+platforms               darwin
+license                 BSD
+maintainers             {@sstallion gmail.com:sstallion} openmaintainer
+
+description             GnuPG PKCS#11 Smart-Card Daemon
+long_description        gnupg-pkcs11 is a project to implement a BSD-licensed           \
+                        smart-card daemon to enable the use of PKCS#11 tokens with      \
+                        GnuPG
+
+homepage                https://github.com/alonbl/gnupg-pkcs11-scd
+master_sites            ${homepage}/releases/download/${name}-${version}
```

Comment:
this should use the [`github` PG](https://guide.macports.org/chunked/reference.portgroup.html#reference.portgroup.github) to simplify the Portfile